Docs: Add documentation with regards to the new prettify function (#36856)
grafana/grafana: docs/sources/panels/visualizations/logs-panel.md

- **Unique labels -** Show or hide the unique labels column, which shows only non-common labels.
- **Common labels -** Show or hide the common labels.
- **Wrap lines -** Toggle line wrapping.

> - **Prettify JSON -** Set this to true if you want to pretty print all JSON logs. This will not affect logs in any format other than JSON.

Reviewer: ```suggestion
- **Prettify JSON -** Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
```
